### PR TITLE
Added the community page

### PR DESCRIPTION
**Pull request by pani2004**
Solved issue #251 

## Description
Added a community page
- [x ] Feature enhancement

## Screenshots / videos (if applicable)

https://github.com/user-attachments/assets/fdb20c24-f2f9-4d29-8394-c315849e8046

## Checklist:
- [x ] I have performed a self-review of my code
- [ x] I have read and followed the Contribution Guidelines.
- [ x] I have tested the changes thoroughly before submitting this pull request.
- [ x] I have provided relevant issue numbers, screenshots, and videos after making the changes.

## Additional context:
 You need to add a Github Api Token to access all those details. GitHub imposes rate limits on how many API requests can be made in an hour. Authenticated requests(the one's with a token) have a higher rate limit compared to unauthenticated requests, allowing you to make more API calls within the allowed timeframe.

### CODE REVIEW

**Comment by pani2004** [comment]
@everly-gif Pls review this PR!!!

File: src/components/community/CommunityPage.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+          <button className="view-contributors-btn" onClick={viewAllContributors}>
+            View All Contributors
+          </button>
+        </div>
+      )}
+    </div>
+  );
+};
+
```
Comment (everly-gif) [inline src/components/community/CommunityPage.jsx:141]: Import the navbar and footer component in the page

File: src/components/community/CommunityPage.jsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+              <p>{repoData.openPRs}</p>
+            </div>
+            <div className="card">
+              <h2>Closed PRs</h2>
+              <p>{repoData.closedPRs}</p>
+            </div>
+            <div className="card">
+              <h2>Contributors</h2>
+              <p>{repoData.contributors}</p>
+            </div>
```
Comment (everly-gif) [inline src/components/community/CommunityPage.jsx:104]: Could you also add a button after this section , saying "contribute in github" or something similar. Before the contributors section.

**Review by everly-gif — COMMENTED** [review]
Everything else looks good functionality wise. Really liked your work, great job!

File: src/components/community/CommunityPage.jsx (hunk 2 above)
Comment (pani2004) [reply src/components/community/CommunityPage.jsx:104]: Will do that

**Comment by pani2004** [comment]
@everly-gif Check now

File: src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -114,6 +114,11 @@ const Footer = () => {
                       <ListItemText primary="Privacy and Policy" />
                     </FooterLink>
                   </ListItem>
+                  <ListItem>
+                    <FooterLink to="/community">
+                      <ListItemText primary="Community Page" />
```
Comment (everly-gif) [inline src/components/Footer/Footer.jsx:119]: Just keep the text as "Community" no need "Page". Also the header appears empty

Comment (pani2004) [reply src/components/Footer/Footer.jsx:119]: @everly-gif I think there is some problem in navbar, in the website also it is not appearing properly pls check it and this happened locally also after I pulled the latest changes.

Comment (pani2004) [reply src/components/Footer/Footer.jsx:119]: Check this - 
https://github.com/user-attachments/assets/58922679-e1c8-4a90-9488-516ccc0926fb

**Comment by pani2004** [comment]
@everly-gif Changed the text

**Review by everly-gif — APPROVED** [review]
Approving your changes! Will open a new issue for header fix so that someone from the community can fix it.

**Comment by everly-gif** [comment]
@Mansi168  pls proceed with final review and merge

**Pull request merged**